google-cloud-clients/google-cloud-datastore/src/main/java/com/google/cloud/datastore/BaseEntity.java

```diff
@@ -631,7 +631,8 @@ public Blob getBlob(String name) {
     return properties.keySet();
   }
 
-  ImmutableSortedMap<String, Value<?>> getProperties() {
+  /** Returns the properties. */
+  public ImmutableSortedMap<String, Value<?>> getProperties() {
```

Comment:
We don't want to expose Guava types as part of the public surface area. We may be able to investigate exposing this as a simple Map type.